sorc/link_workflow.sh

```diff
@@ -151,6 +151,8 @@ done
 
 cd "${HOMEgfs}/scripts" || exit 8
 ${LINK_OR_COPY} "${HOMEgfs}/sorc/ufs_utils.fd/scripts/exemcsfc_global_sfc_prep.sh" .
+${LINK_OR_COPY} "${HOMEgfs}/sorc/gdas.cd/scripts/exglobal_prep_ocean_obs.py"           .
+${LINK_OR_COPY} "${HOMEgfs}/sorc/gdas.cd/scripts/exgdas_global_marine_analysis_ecen.py"           .
```

Comment:
There should be a new (temporary) section for gdasapp script linking similar to what is done for all the executables and parm files:
```suggestion
if [[ -d "${HOMEgfs}/sorc/gdas.cd" ]]; then
  declare -a gdas_scripts=(exglobal_prep_ocean_obs.py \
                           exgdas_global_marine_analysis_ecen.py \
                           )
  for gdas_script in "${gdas_scripts[@]}" ; do
    ${LINK_OR_COPY} "${HOMEgfs}/sorc/gdas.cd/scripts/${gdas_script}" .
  done
fi

Reply:
done